### ISSUE TITLE
Kong locale problem

### ISSUE THREAD

**Issue by pct960** [rbccps-iisc/ideam#42]
The locale settings created by ansible do not persist. Due to this, the `pg_createcluster` fails with  

```perl: warning: Setting locale failed. 
perl: warning: Please check that your locale settings:  
        LANGUAGE = (unset),  
        LC_ALL = (unset),  
        LANG = "en_IN"  
are supported and installed on your system.  
perl: warning: Falling back to the standard locale ("C").  
Error: The locale requested by the environment is invalid.
```

**Comment by harishanand95** [comment]
@pct960 I think this issue of locale was solved with your PR #43.

https://github.com/rbccps-iisc/ideam/blob/master/images/Dockerfile.ubuntu.certified.kong#L11

**Comment by pct960** [comment]
Fixed in https://github.com/rbccps-iisc/ideam/pull/62 for all containers
